src/test/lintError.ts

```diff
@@ -14,6 +14,8 @@
  * limitations under the License.
  */
 
+export { Error } from "../configuration";
```

Comment:
Why does this compile? This import is just an interface, it doesn't define a call or constructor signature. I think L45 is still referencing node's `Error` constructor.